docs/troubleshooting/transferring-your-app.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Issues with transferring the ownership of your application
+description: A guide covering some of the edge cases when transferring your application
```

Comment:
```suggestion
description: Learn how to transfer ownership of your Clerk application
```
I think this makes it a little clearer what this page is about for someone just reading the description in an SEO card for example